Description: On Python 3.3, the __loader__ variable may get set when modules are loaded, so this is ok to override
Author: Michael Droettboom
Origin: upstream, https://github.com/matplotlib/matplotlib/pull/2170
Forwarded: not-needed
Applied-Upstream: https://github.com/matplotlib/matplotlib/commit/263f853d613550476fce560a630d272daec930e5
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/lib/matplotlib/tests/test_basic.py
+++ b/lib/matplotlib/tests/test_basic.py
@@ -16,6 +16,7 @@ def test_override_builtins():
         '__name__',
         '__doc__',
         '__package__',
+        '__loader__',
         'any',
         'all',
         'sum'
